### DISCUSSION TITLE
Is there a way to host a Flask REST API that uses revChatGPT somewhere that supports the Selenium webdriver stuff?

### DISCUSSION THREAD

**Discussion by basketbla** [acheong08/ChatGPT#444] [Q&A] [answered]
I made a project that uses this package. Usually I host things on Render, but I'm not sure if the Cloudflare startup stuff that uses Selenium would work on there. Does anyone have a good solution for this?

**Comment by acheong08** [comment, marked as answer]
You can try https://github.com/acheong08/ChatGPT-lite but the back end is not open source. I am working on an open source version of that.

**Reply by acheong08** [reply]
*Doesn't require browser

**Comment by DiamondDemon669** [comment]
If you wanted to implement this yourself, you could try Xvfb and its python wrappers (look on pypi)

**Comment by acheong08** [comment]
Working on it: https://github.com/acheong08/ChatGPT/blob/main/src/revChatGPT/GPTserver.py
